test/e2e/instrumentation/monitoring/custom_metrics_autoscaling.go

```diff
@@ -0,0 +1,203 @@
+/*
```

Comment:
I believe this test should be under autoscaling, not instrumentation.

Reply:
Moved to autoscaling